client/src/components/Burger/Burger.css

```diff
@@ -0,0 +1,42 @@
+.burger-line-one, .burger-line-two, .burger-line-three {
```

Comment:
I think you need a parent class, like `.burger`

Typically, you'd have something like `.burger` and `.burger--active` to distinguish between the different states.

Reply:
Then you can modify the styling for the burger lines based on whether or not that `.burger--active` class is there